### PR TITLE
Simplify organisation view template

### PR DESCRIPTION
**Pull request by hannalaakso**
## What
Move the non-organisational content into its own template files. Remove a few now redundant conditional checks for whether an organisation is promotional or not. 

## Why
The `show` template was becoming hard to follow partly because of the length of the file and also because of a large number of conditional blocks of logic depending on whether the organisation is promotional not. 

We're currently working on adding a new organisation header to collections so this work is to prep adding some more logic to `organisations/show`. 

⚠️ This repo is Continuously Deployed: make sure you [follow the guidance](https://docs.publishing.service.gov.uk/manual/development-pipeline.html#merge-your-own-pull-request) ⚠️

Follow [these steps](https://guides.rubyonrails.org/upgrading_ruby_on_rails.html) if you are doing a Rails upgrade.

### CODE REVIEW

**Review by chao-xian — CHANGES_REQUESTED** [review]
Nice! Great to see us tidying up the place - thank you @hannalaakso!

Overall this is fine but the naming of files and partials are confusing, long and non-descriptive of what they contain.

I'd suggest naming the partials as:

* `_standard_org_docs_people.html.erb` instead of `_not_promotional_org.html.erb`
* `_standard_org_foi_profiles.html.erb` instead of `_not_promotional_org_2.html.erb`

I also think the `app/views/organisations/_show_content.html.erb` partial is unnecessary and one abstraction level too much, though I can see that it was a good path to follow as part of the process of working out what to split. The name of the partial itself being so similar to `_show.html.erb` suggests it's not necessary. However it would benefit from a rename as well if we kept the partial.

Hope that's ok. Great work so far!

**Comment by hannalaakso** [comment]
Thanks for the review @chao-xian, have a look now. I've renamed the files as suggested. I've now also removed the extra `show_content` partial - I initially added it keeping in mind that we'll be adding more logic to the main page content at a later point but we can always split the main page content out at that point if it feels necessary.

**Review by chao-xian — APPROVED** [review]
Thank you for making those suggested changes! Looks good.
